OWNERS_ALIASES

```diff
@@ -70,6 +70,12 @@ aliases:
     - hogepodge
     - dklyle
     - rjmorse
+  sig-platform-dev-leads:
+    - jzelinskie
+    - enisoc
+    - jzelinskie
+    - enisoc
+    - pwittrock
```

Comment:
As an owner of 4 and initiator of 3 of the 6 subprojects and as a link to Api-Machinery, I officially volunteer to become a lead in this group: https://github.com/enisoc/community/pull/1

Reply:
I'm absolutely in favor of this.

Reply:
Thanks :-)